modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1963,6 +1969,67 @@ public void testGetObjectEntries() throws Exception {
 				"search", "aa"
 			).build(),
 			childObjectEntry1, childObjectEntry2);
+
+		ObjectField objectField = objectFieldLocalService.fetchObjectField(
```

Comment:
What do you think to move this scenario to a private method (**testSearchByRelationshipField** or **testSearchByRelatedObjectEntry**)? or at least put a comment here to give context.